src/improvements/tasks/MultisigTask.sol

```diff
@@ -317,12 +316,23 @@ abstract contract MultisigTask is Test, Script, StateOverrideManager {
         data = abi.encodeWithSignature("aggregate3Value((address,bool,uint256,bytes)[])", calls);
     }
 
-    /// @notice print the data to sig by EOA for single multisig
-    function printEncodedTransactionData(Action[] memory actions) public view {
+    /// @notice Print the data to sign.
+    function printEncodedTransactionData(bytes memory dataToSign) public pure {
         // logs required for using eip712sign binary to sign the data to sign with Ledger
+        console.log("\nData to sign:");
         console.log("vvvvvvvv");
-        console.logBytes(getEncodedTransactionData(parentMultisig, getMulticall3Calldata(actions)));
+        console.logBytes(dataToSign);
         console.log("^^^^^^^^\n");
+
```

Comment:
Not sure if we still want to be printing the data below. Given we're not requiring the use of op-verify these logs might become confusing for signers.